src/GlobalStyles.js

```diff
@@ -2,10 +2,12 @@ import { createGlobalStyle } from 'styled-components'
 
 const GlobalStyles = createGlobalStyle`
 
+
 * {
   box-sizing: border-box;
   padding: 0;
   margin: 0;
+  background-color: #1A1A1D;
```

Comment:
think about setting color variables in GlobalStyles

Reply:
good point :)